doc/manuals/admin/yum.md

```diff
@@ -0,0 +1,47 @@
+## Using Yum repositories
+
+This document describes the guidelines of using FIWARE Yum repository to install Orion Context Broker. Provided configurations are corresponds to x86_64 architecture ($basearch) and CentOS/RHEL 7 OS ($releasever)
+
+There are two available packages:
+- nightly, for RPMs which are built from master branch each night (at around 2am).
+- release, for "official" release RPMs. There isn't a fixed released period although it used to be around a release every 1-2 months.
```

Comment:
Better to say:

The usual release period is 1-2 months.

Reply:
fixed in c6e45d153045d7d51e08a4dc3c732985c22fe5c9